parsec/core/gui/devices_widget.py

```diff
@@ -134,7 +148,13 @@ def on_list_success(self, job):
             )
 
     def on_list_error(self, job):
-        pass
+        status = job.status
+        if status == "error":
+            self._flush_devices_list()
+            self.initialized = False
+            label = QLabel(_("TEXT_USER_LIST_RETRIEVABLE_FAILURE"))
```

Comment:
Should probably be something with `DEVICE_LIST` instead of `USER_LIST`.

Reply:
True, i added this new text entry:

```
TEXT_DEVICE_LIST_RETRIEVABLE_FAILURE
Failed to retrieve the device list from the server.
Impossible de récupérer la liste des appareils depuis le serveur.
```